src/admin-views/help-hub/resources/resources.php

```diff
@@ -9,44 +9,79 @@
+ *
+ * @since TBD
+ *
+ * @param string $description The default resources tab description
+ */
+$description = apply_filters(
+	'tec_help_hub_resources_description',
+	sprintf(
+	/* translators: %1$s is the link to the Knowledgebase. */
+		__( 'Help on setting up, customizing, and troubleshooting your calendar. See our %1$s for in-depth content.', 'tribe-common' ),
+		'<a href="https://evnt.is/1bbw" rel="noopener noreferrer" target="_blank">' . __( 'Knowledgebase', 'tribe-common' ) . '</a>'
+	)
+);
+
+/**
+ * Filter the resources tab notice content
+ *
+ * @since TBD
+ *
+ * @param string $notice_content The default notice content
+ */
+$notice_content = apply_filters(
+	'tec_help_hub_resources_notice',
+	sprintf(
+	// translators: Placeholders are for the `a` tag that displays a link.
+		_x(
+			'To find the answer to all your questions use the %1$sTEC Chatbot%2$s',
+			'The callout notice to try the chatbot with a link to the page',
+			'tribe-common'
+		),
+		'<a data-tab-target="tec-help-tab" href="#">',
+		'</a>'
+	)
+);
 ?>
 <div class="tribe-settings-form form">
 	<div class="tec-settings-form">
 		<div class="tec-settings-form__header-block tec-settings-form__header-block--horizontal">
 			<h2 class="tec-settings-form__section-header">
-				<?php
-				echo esc_html_x( 'Resources', 'Resources tab title', 'tribe-common' );
-				?>
+				<?php echo esc_html( $hub_title ); ?>
 			</h2>
 			<p class="tec-settings-form__section-description">
-				<?php
-				printf(
-				/* translators: %1$s is the link to the Knowledgebase. */
-					esc_html__( 'Help on setting up, customizing, and troubleshooting your calendar. See our %1$s for in-depth content.', 'tribe-common' ),
-					'<a href="https://evnt.is/1bbw" rel="noopener noreferrer" target="_blank">' . esc_html__( 'Knowledgebase', 'tribe-common' ) . '</a>'
-				);
-				?>
+				<?php echo wp_kses_post( $description ); ?>
 			</p>
 			<?php
-			$notice_content = sprintf(
-			// translators: Placeholders are for the `a` tag that displays a link.
-				_x(
-					'To find the answer to all your questions use the %1$sTEC Chatbot%2$s',
-					'The callout notice to try the chatbot with a link to the page',
-					'tribe-common'
-				),
-				'<a data-tab-target="tec-help-tab" href="#">',
-				'</a>'
-			);
-
 			echo wp_kses( $help_hub->generate_notice_html( $notice_content, 'tec-common-help-chatbot-notice' ), 'post' );
 
 			?>
 		</div>
-		<?php $this->template( 'help-hub/resources/getting-started' ); ?>
-		<?php $this->template( 'help-hub/resources/customization' ); ?>
-		<?php $this->template( 'help-hub/resources/common-issues' ); ?>
-		<?php $this->template( 'help-hub/resources/faqs' ); ?>
+		<?php
+		foreach ( $sections as $slug => $section ) {
+			$template_type = $section['type'] ?? 'default';
+			$template_name = $template_map[ $template_type ] ?? $template_map['default'];
+
+			$this->template( $template_name, [ 'section' => $section ] );
+		}
+		?>
 
 		<div class="tec-settings-infobox">
```

Comment:
Don't know if it is, but this feels like something that will get used in several places, so it might warrant a template of its own. (The Chatbot infobox)